### ISSUE TITLE
TypeError: Cannot read property 'parse' of undefined on macOS Big Sur

### ISSUE THREAD

**Issue by matteobertoldo** [bali182/autocomplete-json#84]
[Enter steps to reproduce:]

1. Install `autocomplete-json` package from "Install Packages" UI. 
2. Wait until the installation is finished.

**Atom**: 1.53.0 x64
**Electron**: 6.1.12
**OS**: macOS 11.0.1
**Thrown From**: [autocomplete-json](https://github.com/bali182/autocomplete-json) package 5.5.3

### Stack Trace

Failed to activate the autocomplete-json package

```
At Cannot read property 'parse' of undefined

TypeError: Cannot read property 'parse' of undefined
    at /packages/autocomplete-json/src/json-schema-loader.js:40:64
    at memoized (/packages/autocomplete-json/node_modules/lodash/memoize.js:62:23)
    at resolve (/packages/autocomplete-json/src/json-schema-resolver.js:79:31)
    at Function.createFromProvider (/packages/autocomplete-json/src/json-schema-proposal-provider.js:23:12)
    at /packages/autocomplete-json/src/main.js:50:42
    at Array.map (<anonymous>)
    at Object.consumeJsonSchemaProviders (/packages/autocomplete-json/src/main.js:50:6)
    at Provider.provide (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1107020)
    at ServiceHub.consume (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3564546)
    at Package.activateServices (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3572831)
    at Package.activateNow (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3569586)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:2607
    at Function.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1190051)
    at Emitter.emit (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1191492)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:383812
    at Set.forEach (<anonymous>)
    at PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:383774)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:916621
    at r (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:911227)
    at s (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1134638)
    at ChildProcess.<anonymous> (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1134844)
    at ChildProcess.emit (events.js:200:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
```

### Commands

```
     -3:19.5.0 settings-view:open (atom-pane.pane)
     -2:04.1.0 editor:newline (input.hidden-input)
  6x -1:59.4.0 core:backspace (input.hidden-input)
```

### Non-Core Packages

```
atom-ide-ui 0.13.0 
atom-inline-blame 0.0.10 
atomforce 0.12.0 
autocomplete-json 5.5.3 
autocomplete-modules 2.3.0 
busy-signal 2.0.1 
color-picker 2.3.0 
editorconfig 2.6.1 
file-icons 2.1.44 
file-types 1.0.1 
flex-tool-bar 2.2.1 
font-viewer 0.4.0 
highlight-selected 0.17.0 
hyperlink-hyperclick 3.0.1 
ide-typescript 0.9.1 
intentions 1.1.5 
language-babel 2.85.0 
language-gitattributes 1.0.5 
language-htaccess 0.2.0 
language-ignore 1.8.0 
language-markdown 0.37.0 
language-mjml 2.1.0 
language-sfcc 0.6.1 
linter 3.1.1 
linter-eslint 8.5.5 
linter-htmlhint 1.6.3 
linter-js-yaml 1.4.6 
linter-jsonlint 1.4.0 
linter-markdown 5.2.11 
linter-shellcheck 1.6.0 
linter-stylelint 4.5.1 
linter-tslint 1.10.43 
linter-ui-default 2.1.5 
linter-xmllint 1.4.4 
minimap 4.35.2 
minimap-autohider 1.6.0 
pdf-view 0.72.0 
prettier-atom 0.60.1 
project-config 1.0.2 
project-manager 3.3.8 
smart-html-tags 1.2.5 
split-diff 1.6.1 
svg-preview 0.14.0 
sync-settings 5.0.8 
teletype 0.13.4 
tool-bar 1.4.2 
```

**Comment by UziTech** [comment]
This must be an apm issue. I don't think `urlJs` can be undefined with `import uriJs from 'uri-js'`

Try uninstalling it and re installing it.

**Comment by UziTech** [comment]
I found the issue. this should be fixed in v5.5.4
